src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -25,14 +25,18 @@
 import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.appengine.api.datastore.Query.SortDirection;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 import java.util.UUID;
 
 /** Provides access to the data stored in Datastore. */
 public class Datastore {
 
-  private DatastoreService datastore;
-
+  private DatastoreService datastore; 
+  private static int longestMessage = 0;
+  private static HashMap<String, Integer> users = new HashMap<String,Integer>();
```

Comment:
users seems a bit confusing as a name here, since users.get('name') returns an int, not a user object.

Perhaps postsPerUser or something could be a more clear name?